fastchat/serve/router/router_processer.py

```diff
@@ -126,10 +126,10 @@ def pack_message(self, message: str) -> str:
     async def process(self) -> Generator[str, Any, None]:
         if not self.isRAG:
```

Comment:
Do you know if we still need to check these two lines?
@roy-shih

Reply:
Or can we rephrase it like before?

```python
if self.isRAG.RAG == False:
    return

yield self.pack_message(f"""<SYSTEM>{
        {"task":"RAG","params":{"message": f"start {self.app['AgentName']} RAG"}}
    }</SYSTEM>""")
```

Reply:
Currently, we will send the isRAG parameter from the chat room, but it does not work in the Langchain case. So I recommend keeping it.
isRAG has three parameters: RAG, API, Recommend. 
If isRAG.RAG is true, it will be sent to the RAG server. 
If isRAG.API or isRAG.Recommend is true, there will be additional processing, which has not been designed yet.

Reply:
Got it.